specification/sql/resource-manager/readme.md

```diff
@@ -24,125 +24,13 @@ These are the global settings for the Sql API.
 title: SqlManagementClient
 description: 'The Azure SQL Database management API provides a RESTful set of web services that interact with Azure SQL Database services to manage your databases. The API enables you to create, retrieve, update, and delete databases.'
 openapi-type: arm
-tag: package-preview-2021-02
```

Comment:
@akning-ms 
I made this change and lead to tons of failed checks. 
the master version is: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/sql/resource-manager/readme.md

do you think this is an issue when we use OpenAPI to create a new branch?

Reply:
@zhenglaizhang  to help take a look openapi hub

Reply:
Hi @ericshape as you used openapi hub to create the preview-2021-02 version, and openapi hub will update the readme default tag as preview-2021-02, this is by design and also suggested practice to copy all swaggers from old versions to latest new version, and make it self contained.
I notice that package-composite-v5 tag exists mainly for .net v5 client compatibility issues, so in future if these is required, you can always rely on the openapi hub's current behavior and no need to revert the default tag updating by hub.